### ISSUE TITLE
Add support for Go Modules

### ISSUE THREAD

**Issue by kamilogorek** [getsentry/raven-go#236]
I couldn't find any reliable way to extract current package name, as in here: https://github.com/getsentry/raven-go/blob/238ebd86338d2e91cb5c84663d695877dd5d0ff4/stacktrace_test.go#L109-L135 due to Go Modules not using GOPATH anymore.
Need more investigation.

Some sources:
https://github.com/golang/go/wiki/Modules#migrating-to-modules
https://research.swtch.com/vgo-module

**Comment by tankbusta** [comment]
Is this issue why I'm seeing the raven package in the error title? (We're using modules)

![theerror](https://user-images.githubusercontent.com/592749/52368328-604a4e00-2a0b-11e9-849f-a03abc672815.PNG)

**Comment by kamilogorek** [comment]
I don't think so, I was able to successfully read the module name using modules as well.

Do you have any repro of this behaviour?
